src/store/root.store.ts

```diff
@@ -1,10 +1,12 @@
-import { rootReducer } from './root.reducer';
+import rootReducer from './reducers/root.reducer';
 import { saga } from './root.saga';
 import { createStore, applyMiddleware } from 'redux';
 import { watchPets } from './sagas/comparison.saga';
+import {watchLoadPet} from './sagas/pet.saga';
 import {composeWithDevTools} from 'redux-devtools-extension';
 
 export const store = createStore(rootReducer, composeWithDevTools( applyMiddleware(saga) ));
 
 saga.run(watchPets);
+saga.run(watchLoadPet);
```

Comment:
It is bad practice to put all watchers inside the root file (P.S. as well you need to put it inside `root.saga.ts` file instead of `root.store.ts`)

Please contact @IvanBonk to improve this part of code. I showed him the better way of structuring sagas.

Reply:
done